### PR TITLE
Add additional Snackbar WinUI notes

### PR DESCRIPTION
**Pull request by VladislavAntonyuk**
Add additional WinUI specifics. Inspired by https://github.com/CommunityToolkit/Maui/issues/353

### CODE REVIEW

**Comment by PRMerger13** [comment]
## PRMerger Results
 | Issue | Description |
 --- | --- |
 File Change Percent | This PR contains file(s) with more than 20% file change.

File: docs/maui/alerts/snackbar.md [hunk 1]
```diff
@@ -116,4 +116,104 @@ You can find the source code for `Snackbar` over on the [.NET MAUI Community Too
...
 2. "Native" Snackbar is available only on Android and created by Google. Other platforms use "Container" (`UIView` for iOS and MacCatalyst, `ToastNotification` on Windows).
-3. `Snackbar` on Windows can't be anchored to `VisualElement` and is always displayed as a default Windows Notification.
\ No newline at end of file
+3. `Snackbar` on Windows can't be anchored to `VisualElement` and is always displayed as a default Windows Notification.
+
+### WinUI specifics
+
+`ToastNotification` which is used to show `Snackbar` on Windows has 2 types of activation: foreground and background.
+
+Foreground activation type is used in `CommunityToolkit.Maui` library. That means the new instance of application is executed. It is up to the developer how to handle such situation. There are some proposals:
```
Comment (jfversluis) [inline docs/maui/alerts/snackbar.md:125]:
```suggestion
Foreground activation type is used in `CommunityToolkit.Maui` library. That means, whenever a notification is shown a new instance of application is executed. It is up to the developer how to handle such situations. Here are a few suggestions:
```

I don't know anything about this, so please check if what I corrected makes any sense :D also do we have a link to some documentation on the foreground and background activation stuff?

File: docs/maui/alerts/snackbar.md [hunk 2]
```diff
@@ -116,4 +116,104 @@ You can find the source code for `Snackbar` over on the [.NET MAUI Community Too
...
+
+1. Use Single Application Instance.
```
Comment (jfversluis) [inline docs/maui/alerts/snackbar.md:127]: Could we add a little explanation of what this should do? If you're not comfortable writing it explain to me what it does and I'll write it up :)

This means that whenever a notification is shown, it checks if there is already a process running for our app and it kills the new process that was created by the notification?

File: docs/maui/alerts/snackbar.md [hunk 3]
```diff
@@ -116,4 +116,104 @@ You can find the source code for `Snackbar` over on the [.NET MAUI Community Too
...
+    }
+    catch (AbandonedMutexException)
+    {
+        mutex.ReleaseMutex();
+        return mutex.WaitOne(0, false);
+    }
+}
+```
+
+2. Using external NuGet package.
```
Comment (jfversluis) [inline docs/maui/alerts/snackbar.md:164]: Same for this one. If I use this method, what will it do?

**Review by jfversluis — CHANGES_REQUESTED** [review]

**Comment by PRMerger20** [comment]
## PRMerger Results
 | Issue | Description |
 --- | --- |
 File Change Percent | This PR contains file(s) with more than 20% file change.

**Comment by VladislavAntonyuk** [comment]
@jfversluis , could you suggest how to pass the link to the external docs?

**Comment by jfversluis** [comment]
@davidbritch ? Do you have some advice for this? :)

**Comment by bijington** [comment]
> @davidbritch ? Do you have some advice for this? :)

I provided a suggestion for Vlad. Basically we remove the docs.Microsoft.com part so the url is relative and then we make sure to exclude any locale information

**Comment by bijington** [comment]
The details are currently sat in this PR: https://github.com/MicrosoftDocs/CommunityToolkit/pull/30

I guess some of those details may want to be in the shared guide but I do find it easier to read everything from a single place

**Comment by PRMerger7** [comment]
## PRMerger Results
 | Issue | Description |
 --- | --- |
 File Change Percent | This PR contains file(s) with more than 20% file change.

**Comment by PRMerger3** [comment]
## PRMerger Results
 | Issue | Description |
 --- | --- |
 File Change Percent | This PR contains file(s) with more than 20% file change.

**Comment by PRMerger5** [comment]
## PRMerger Results
 | Issue | Description |
 --- | --- |
 File Change Percent | This PR contains file(s) with more than 20% file change.

**Review by jfversluis — APPROVED** [review]

**Pull request merged**